wa/framework/uiauto/app/src/main/java/com/arm/wa/uiauto/BaseUiAutomation.java

```diff
@@ -173,6 +215,108 @@ public int getDisplayCentreHeight() {
         return getDisplayHeight() / 2;
     }
 
+    public void tapDisplayCentre() {
+        tapDisplay(getDisplayCentreWidth(),  getDisplayCentreHeight());
+    }
+
+    public void tapDisplay(int x, int y) {
+        mDevice.click(x, y);
+    }
+
+    public void pressEnter() {
+        mDevice.pressEnter();
+    }
+
+    public void pressHome() {
+        mDevice.pressHome();
+    }
+
+    public void pressBack() {
+        mDevice.pressBack();
+    }
+
+    public void uiObjectSwipe(UiObject view, Direction direction, int steps) throws Exception {
+        switch (direction) {
+            case UP:
+                view.swipeUp(steps);
+                break;
+            case DOWN:
+                view.swipeDown(steps);
+                break;
+            case LEFT:
+                view.swipeLeft(steps);
+                break;
+            case RIGHT:
+                view.swipeRight(steps);
+                break;
+            case NULL:
+                throw new Exception("No direction specified");
+            default:
+                break;
+        }
+    }
+
+        public void uiObjectVertPinchIn(UiObject view, int steps, int percent) throws Exception {
```

Comment:
Indentation level seems to change from this line onwards.

Reply:
I can't seem to see the change in indention. Has this been fixed?

Reply:
yup, seems to be fixed now. :+1: